functions/callback-functions.md

```diff
@@ -0,0 +1,214 @@
+# Callback Functions
+
+## Goal
+
+One of the ways we'll solve problems in JavaScript is using callback functions. Getting the hang of callback functions takes practice.
+
+The goal of this lesson is to introduce callback functions, so we can use this vocabulary as we see functions passed around as arguments in the future.
+
+## Another Way to Look at Functions
+
+We might have been in situations before where we thought, "Ugh! This logic is reusable. The only thing that needs to change is one piece of behavior." If we've ever thought, "If only we could take in behavior as an argument," we're in luck!
+
+For example, let's imagine this problem and pseudocode:
+
+We need to build one calculator function that can add, subtract, multiply, and divide two numbers.
+
+In order to accomplish that goal, we need three arguments:
+
+1. The first number
+2. The second number
+3. Something that represents to operation we need, to help us determine if we're adding, subtracting, multiplying, or dividing
+
+We could approach the third argument by expecting a string, such as `'add'`, `'+'`, `'multiply'`, or `'*'`.
```

Comment:
Be a bit more explicit about our decision and how it might be handled
```suggestion
We could decide that the third argument will expect a string, such as `'add'`, `'+'`, `'multiply'`, or `'*'`. Our `calculate` function could inspect the string and decide which behavior to apply.
```